### PR TITLE
Stubs for Versioning 2

### PR DESCRIPTION
**Pull request by ShahabT**
## What changed?

Import latest API with Versioning 2 changes and make server code compilable with it.

## Why?

Preparing to merge versioning-2 feature branch after.

## How did you test it?

None

## Potential risks

None

## Documentation

None

## Is hotfix candidate?

No

### CODE REVIEW

File: api/adminservice/v1/request_response.pb.go
Comment (dnr) [inline api/adminservice/v1/request_response.pb.go:1]: It looks like these proto files are generated without the utf-8 stuff.. did you update all the proto tools?

Comment (ShahabT) [reply api/adminservice/v1/request_response.pb.go:1]: I updated the tools now, please check again if the protos are generated correctly

File: common/api/metadata.go [hunk 1]
```diff
@@ -127,6 +127,8 @@ var (
 		"ListSchedules":                      {Scope: ScopeNamespace, Access: AccessReadOnly},
 		"UpdateWorkerBuildIdCompatibility":   {Scope: ScopeNamespace, Access: AccessWrite},
 		"GetWorkerBuildIdCompatibility":      {Scope: ScopeNamespace, Access: AccessReadOnly},
+		"UpdateWorkerVersioningRules":   	  {Scope: ScopeNamespace, Access: AccessWrite},
```
Comment (dnr) [inline common/api/metadata.go:130]: can you just fix this indentation? (run gofmt/goimports on it or just open/save in your editor probably does it)

(and other files too)

File: service/frontend/configs/quotas.go [hunk 2]
```diff
@@ -106,6 +106,7 @@ var (
 		"/temporal.api.workflowservice.v1.WorkflowService/DescribeWorkflowExecution":     3,
 		"/temporal.api.workflowservice.v1.WorkflowService/DescribeTaskQueue":             3,
```
Comment (dnr) [inline service/frontend/configs/quotas.go:107]: what's the plan here? do we need to use a different rate limit here if it requests reachability (or always)?

Comment (ShahabT) [reply service/frontend/configs/quotas.go:107]: No DescribeTaskQueue will be subject to Visibility limit regardless of reachability requested or not, until we support content-based rate limiting.

**Review by dnr — APPROVED** [review]

**Pull request merged**